helm/hpcc/templates/_helpers.tpl

```diff
@@ -618,7 +620,19 @@ Check dalistorage mount point, using hpcc.changeMountPerms
 */}}
 {{- define "hpcc.checkDaliMount" -}}
 {{- if .root.Values.storage.daliStorage.forcePermissions | default false }}
-{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" "dalistorage" "volumePath" "/var/lib/HPCCSystems/dalistorage") }}
+{{- $volumeName := (.root.Values.storage.daliStorage.plane | default "dalistorage") -}}
+{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" $volumeName "volumePath" "/var/lib/HPCCSystems/dalistorage") }}
+{{- end }}
+{{- end }}
+
+{{/*
+Check sashastorage mount point, using hpcc.changeMountPerms
+Pass in sashaName
+*/}}
+{{- define "hpcc.checkSashaMount" -}}
+{{- if .me.storage.forcePermissions | default false }}
+{{- $volumeName := (.me.storage.plane | default "sashastorage") -}}
+{{ include "hpcc.changeMountPerms" (dict "root" .root "volumeName" $volumeName "volumePath" "/var/lib/HPCCSystems/sasha") }}
```

Comment:
the volumePath won't be right, if the plane has changed the prefix (NB: can be >1 sasha plane, different services using a different plane)

Reply:
I replace with hpcc.getVolumeMountPrefix

Reply:
This isn't directly related to the issue of this NFS, it's an issue in general for anything that needs forcePermissions.
See other comment to re. complication with initContainers and sasha services.

It would be better if you fixed these issues in a JIRA/PR 1st (unrelated to NFS example chart).